src/components/print/PrintDirective.js

```diff
@@ -659,7 +659,8 @@
 
       // Transform layers to literal
       angular.forEach(layers, function(layer) {
-        if (layer.visible) {
+        if (layer.visible && (!layer.timeEnabled ||
+               (layer.timeEnabled && layer.time != undefined))) {
```

Comment:
testing `layer.timeEnabled` is not needed because you already test `!layer.timeEnabled` above, am I wrong ?